content/partials/types/_push_channel.textile

```diff
@@ -41,9 +41,9 @@ h6(#list-subscriptions).
 
 bq(definition).
   java,android: "PaginatedResult":#paginated-result<"PushChannelSubscription":#push-channel-subscription> listSubscriptions(String deviceId, String clientId, String deviceClientId, String channel)
-  objc,swift: listSubscriptions(deviceId: String?, clientId: String?, deviceClientId: String?, channel: String?, callback: ("ARTPaginatedResult":#paginated-result<"PushChannelSubscription":#push-channel-subscription>?, ARTErrorInfo?) -> Void)
+  objc,swift: listSubscriptions(deviceId: String?, clientId: String?, deviceClientId: String?, `channel: String?, callback: ("ARTPaginatedResult":#paginated-result<"PushChannelSubscription":#push-channel-subscription>?, ARTErrorInfo?) -> Void)
 
-"Unsubscribe all devices associated with your device's clientId":/general/push/activate-subscribe#subscribing-client-id from the channel's push notifications.
+Lists push subscriptions on a channel specified by it's channel name (@channel@). These subscriptions can be either client (@clientId@) subscriptions, device (@deviceId@) subscriptions, or if @concatFilters@ is set to @true@, both. This method requires clients to have the "Push Admin capability":/general/push/admin. For more information, take a look at the @GET rest.ably.io/push/channelSubscriptions@ "Rest API":/rest-api.
```

Comment:
```suggestion
Lists push subscriptions on the specified channel (@channel@). These subscriptions are either client (@clientId@) subscriptions, device (@deviceId@) subscriptions, or if @concatFilters@ is set to @true@, both. This method requires clients to have the "Push Admin capability":/general/push/admin. For more information, see the @GET rest.ably.io/push/channelSubscriptions@ "Rest API":/rest-api.
```

Reply:
(it's -> its)

Reply:
I did some of those things on purpose. 😅 e.g.  
- `a channel specified by it's channel name (@channel@)`. The argument is the channel name, the entity is a channel. 
- `These subscriptions can be either`. There is no such thing as a subscription that is simultaneously both clientId and deviceId. _**They can be either**_. 🙂. Your suggestion makes it seem like a subscription can be both. 
- Ill change `take a look at` to `see`, but I'm not sure if it makes a difference here (happy to be convinced though 🤓).

Reply:
I've elaborated the original text for point 1 above: ea5363f58a472b753ed2a510b003a2af00ee02c6
I've fixed the 3rd and 4th point above: b5003df7b537c6f11de77159b41c7427a1d03395